multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -691,6 +693,13 @@ def _setup_distillation(
         else:
             raise ValueError(f"Unknown soft_label_loss_type: {self._config.distiller.soft_label_loss_type}")
 
+        if self._config.distiller.intermediate_loss_type == "cos":
```

Comment:
`cos` seems not straightforward. How about `cosine`?

Reply:
Done.